webui/react/src/components/MetricBadgeTag.tsx

```diff
@@ -10,9 +11,7 @@ interface Props {
 
 const MetricBadgeTag: React.FC<Props> = ({ metric }: Props) => {
   return (
-    <BadgeTag label={metric.name} tooltip={metric.type}>
-      {metric.type.substr(0, 1).toUpperCase()}
-    </BadgeTag>
+    <BadgeTag label={metric.name} tooltip={metric.type}>{capitalize(metric.type)}</BadgeTag>
```

Comment:
`capitalize(metric.type)` returns the whole word, while the previous `metric.type.substr(0, 1).toUpperCase()` just returned the first letter. Since `substr` is deprecated, I suggest changing this line to `metric.type.slice(0, 1).toUpperCase()`, which maintains previous behavior.